src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -950,6 +950,9 @@ private void processCandidateData() {
       // catch that and report a helpful error. Validation also hits this code path, though, so we
       // need to prevent a NullPointerException here.
       if (getRandomSeed() != null) {
+	    // sort candidate permutation for reproducibility
+	    Collections.sort(candidatePermutation);
```

Comment:
This seems reasonable. Is it actually possible for this to be in a different order, though?

Also, indentation is off.

Reply:
In the current code, i don't think so.  But this hook helps us define the input to shuffle() in case other processing changes at some point.